index.js

```diff
@@ -23,31 +23,16 @@ function validatePaymentDataMiddleWare(req, res, next) {
     next();
 };
 
-function getAccountBalanceData(inPayments) {
-    const inPaymentsAmount = inPayments.map(payment => payment.amount);
-    const homeAmount = inPaymentsAmount.reduce((a, b) => a + b, 0);
-    const homeCurrency = 'GBP';
-    const homeCurrencySymbol = '\u00A3';
-
-    const balance = {
-        "amount": homeAmount,
-        "currency": homeCurrency,
-        "currencySymbol": homeCurrencySymbol,
-    };
-    return balance;
-};
-
 server.get('/payments/balance', (req, res) => {
-    const balance = getAccountBalanceData(paymentsIn);
     res.status(200).send(balance);
```

Comment:
hm, this is not the best way either.
well, actually having `getAccountBalanceData(payments)` would be ok,
but, what wasn't ok are these 2 lines: 
```js
    const inPaymentsAmount = inPayments.map(payment => payment.amount);
    const homeAmount = inPaymentsAmount.reduce((a, b) => a + b, 0);
```
they are calculating rubbish, taking in account that our balance can be topped up in multicurrency.
say, your home currency is GBP, you have some money in EUR, but for your trip to Spain you need more, 
so borrow a little bit from your close friend at some exch.Rate = x, that's your debt but from the balance prospective it is an income payment in EUR but you convert it GBP coz you need to know Totals. 
2 weeks later you come back and think of returning your debt. But spent all your EUR cash, so you take some money from the Debit Card in GBP and convert it into EUR at the exch.Rate = z.
get the idea?

that's why we need to account incoming payments in different currencies 
and that's why those 2 lines are not suitable for us.
but, calculating balance now should be a simple function, we don't need a whole component for it.